src/pages/ReportAvatar.tsx

```diff
@@ -23,19 +23,18 @@ type ReportAvatarProps = ReportAvatarOnyxProps & StackScreenProps<AuthScreensPar
 function ReportAvatar({report = {} as Report, policies, isLoadingApp = true, route}: ReportAvatarProps) {
     const policy = policies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID ?? '0'}`];
     const title = policy ? ReportUtils.getPolicyName(report, false, policy) : report?.reportName;
-    let avatarURL = undefined;
-    let fileName = undefined;
-    let groupChatDraft;
+    let avatarURL;
+    let fileName;
+    const [groupChatDraft] = useOnyx(ONYXKEYS.NEW_GROUP_CHAT_DRAFT);
```

Comment:
```suggestion
    const shouldUseGroupChatDraft = !!route.params.isNewGroupChat;
    const [groupChatDraft] = useOnyx(ONYXKEYS.NEW_GROUP_CHAT_DRAFT, {initWithStoredValues: shouldUseGroupChatDraft});
```

A little optimization, we don't need to get onyx values if we are not going to use it